test/staking/DonationsStaking.test.ts

```diff
@@ -281,13 +279,9 @@ describe("DonationsStaking - DonationStaking contract that receives funds in ETH
     const ethBalanceAfterWithdraw = await donationsStaking.provider.getBalance(
       donationsStaking.address
     );
-    const avatarETHBalanceAfterWithdraw =
-      await donationsStaking.provider.getBalance(avatar);
     expect(totalStakedAfterEnd).to.be.equal(0);
-    expect(ethBalanceAfterWithdraw).to.be.equal(0);
-    expect(avatarETHBalanceAfterWithdraw).to.be.equal(
-      ethBalanceBeforeWithdraw.add(avatarETHBalanceBeforeWithdraw)
-    );
+    expect(ethBalanceBeforeWithdraw).to.be.gt(0);
+    expect(ethBalanceAfterWithdraw).to.be.equal(ethBalanceBeforeWithdraw);
```

Comment:
makes absolutly no sense, you are testing the TRIVIAL CASE!

Reply:
ok i think i was wrong here, and you didnt double check me
this is the withdraw method test case, i thought it was a test, testing the withdraw when switching contract
the withdraw method leaves the eth.
so
- actually change withdraw to send also eth to avatar